rules.html

```diff
@@ -1,39 +1,63 @@
-        <p> 
-        There are 3 distinct sub-challenges. Each sub-challenge is managed 
-        through its individual Codabench page&mdash;combined results will be listed 
-        on the leaderboard page here. You may participate in 
-        each separate out of domain challenge, or any combination of 
-        sub-challenges. To participate in the <i>shared challenge</i>, you must 
-        participate in all 3 with the same team name. The winner of the 
-        combined challenge will be determined by taking the smallest sum 
-        of each team's individual challenge ranking, summed over the 3 
-        separate challenges. For example, 1st place in challenge A, 2nd in 
-        B, and 4th place in challenge C would give you a score of 7, the 
-        smallest summed score wins!
-        </p>
-        <p>
-      We will announce the winners here at the end of the challenge, and invite them to
-      the winners' ceremony at the ML Challenge workshop in Spring 2026. 
-      </p>
+    <div id="header"></div>
+
+    <section class="hero">
+        <div id="logo"></div>
+        <div class="hero-content" id="leaderboard-content">
+            <h2>How to participate</h2>
+            <p>
+                There are 3 distinct sub-challenges. Each sub-challenge is managed
+                through its individual Codabench page&mdash;combined results will be listed
+                on the leaderboard page here. You may participate in
+                each separate out of domain challenge, or any combination of
+                sub-challenges. To participate in the <i>shared challenge</i>, you must
+                participate in all 3 with the same team name. The winner of the
+                combined challenge will be determined by taking the smallest sum
+                of each team's individual challenge ranking, summed over the 3
+                separate challenges. For example, 1st place in challenge A, 2nd in
+                B, and 4th place in challenge C would give you a score of 7, the
+                smallest summed score wins!
+            </p>
+            <p>
+                We will announce the winners here at the end of the challenge, and invite them to
+                the winners' ceremony at the ML Challenge workshop in Spring 2026.
+            </p>
+        </div>
+    </section>
+
+    <div class="image-row">
+        <a href="https://www.codabench.org/competitions/9854/" target="_blank" class="circle" id="circle1">
+            <img src="images/beetles.png" alt="Beetles">
+            <div class="hover-text" id="hover-text1">
+                Beetles as Sentinel Taxa
+            </div>
+        </a>
+        <a href="https://www.codabench.org/competitions/9806/" target="_blank" class="circle" id="circle2">
+            <img src="images/neural_new.png" alt="neural">
+            <div class="hover-text" id="hover-text2">
+                Forecasting Neural activity from Monkeys brain
```

Comment:
```suggestion
                Forecasting Monkey Motor Neuron behavior
```
Title in the slides. Or just add an apostrophe